juicy-html.html

```diff
@@ -55,7 +55,7 @@
                 },
                 'href': {
                     set: function(newValue) {
-                        if(newValue === null){
+                        if(newValue === null || newValue === undefined){
```

Comment:
Please format the code according to our guides, which is default Visual Studio formatting.

```js
if(newValue === null || newValue === undefined){
```

Should be:

```js
if (newValue === null || newValue === undefined) {
```